api/v1alpha1/kbsconfig_types.go

```diff
@@ -76,6 +76,9 @@ type KbsConfigSpec struct {
 
 	// kbsResourcePolicyConfigMapName is the name of the configmap that contains the Resource Policy
 	KbsResourcePolicyConfigMapName string `json:"kbsResourcePolicyConfigMapName,omitempty"`
+
+	// kbsTdxConfigMapName is the name of the configmap that contains the TDX specific config
+	KbsTdxConfigMapName string `json:"kbsTdxConfigMapName,omitempty"`
```

Comment:
Should we create a separate struct for TDX config? In case there are other configs that might be needed going forward